server/controller/src/main/java/ai/starwhale/mlops/api/DataStoreController.java

```diff
@@ -191,6 +192,7 @@ public void scanAndExport(ScanTableRequest request, HttpServletResponse httpResp
                             request.getTables().stream().map(TableDesc::getTableName).collect(
                                     Collectors.toList())) + ".csv\"");
             httpResponse.addHeader("Content-Length", String.valueOf(bytes.length));
+            httpResponse.addHeader("Content-Type", "application/octet-stream;charset=utf-8");
```

Comment:
Would it be better to use `Content-Type: text/csv;charset=utf-8`?
https://datatracker.ietf.org/doc/html/rfc4180